src/components/skeleton/_skeleton-placeholder.scss

```diff
@@ -0,0 +1,9 @@
+@import '../../globals/scss/helper-mixins';
+@import '../../globals/scss/import-once';
+@import '../../globals/scss/vars';
+
+@include exports('skeleton-placeholder') {
+  .#{$prefix}--skeleton__placeholder {
+    @include skeleton;
```

Comment:
Can we add `width`/`height` styles so the React variant can be used without in-line styles?

@alisonjoseph Does skeleton support `rem`-based dimensions?

Reply:
@asudoh Could you expound on that? I agree that we should probably avoid in-line styles, but I'm curious what the alternative implementation might look like.

Reply:
@ryanomackey Great that you asked - We can do something like:

```scss
.#{$prefix}--skeleton__placeholder {
  @include skeleton;
  width: rem(50px);
  height: rem(50px);
}
```

And consumers should be able to:

```scss
.bx--skeleton__placeholder.my--skeleton__placeholder {
  width: 100px;
  height: 100px;
}
```

And apply `my--skeleton__placeholder` class to the React component, etc. Let me know if you have any further questions. Thanks!

Reply:
Sorry for my late reply on this.  Yes, you can use rem for height/width. 

For the React component were you planning on setting the height/width there as props? Still wouldn't hurt to have a default in there as Akira is suggestings. Then a user could override with either css or the React component could override with inline css.

Reply:
@asudoh @alisonjoseph Okay, so I added some default dimensions to the class, and then over in the [React PR](https://github.com/IBM/carbon-components-react/pull/1401), I got rid of the inline styles and added an example with custom classes to the storybook.

So, consumers can still use inline styles, but this should nudge them in a more performant direction 😉